cmd/weaver-kube/babysitter.go

```diff
@@ -30,15 +36,61 @@ var babysitterCmd = tool.Command{
 	Flags:       babysitterFlags,
 	Description: "The weaver kubernetes babysitter",
 	Help: `Usage:
-  weaver kube babysitter
+  weaver kube babysitter <weaver config file> <babysitter config file> <component>...
 
 Flags:
   -h, --help   Print this help message.`,
 	Fn: func(ctx context.Context, args []string) error {
-		if len(args) != 0 {
-			return fmt.Errorf("usage: weaver kube babysitter")
+		// Parse command line arguments.
+		if len(args) < 3 {
+			return fmt.Errorf("want >= 3 arguments, got %d", len(args))
 		}
-		return impl.RunBabysitter(ctx)
+		app, err := parseWeaverConfig(args[0])
+		if err != nil {
+			return err
+		}
+		config, err := parseBabysitterConfig(args[1])
+		if err != nil {
+			return err
+		}
+		components := args[2:]
```

Comment:
just a clarification question: does args >= 3 mean `weaver config`, `babysitter config` and at least one component? I know when you run a normal binary the first argument is the exec name.

Reply:
> just a clarification question: does args >= 3 mean weaver config, babysitter config and at least one component?

Yup! For these commands, the binary name is already removed.